### DISCUSSION TITLE
Consider supporting a subdirectory in url for self-hosted

### DISCUSSION THREAD

**Discussion by aureq** [simple-login/app#2078] [Feature Requests]
It would be great if simple-login was able to run under a specific subdirectory instead of `/`. This is relevant since I use kubernetes/docker and host all my home services under the same domain (`home.example.net`). This would allow me to run simple-login like this `https://home.example.net/simple-login`

Thank you

**Comment by leor-gh** [comment]
There is an issue https://github.com/simple-login/app/issues/1952 opened against supporting this use case.

TL;DR, on the big picture, it is not that difficult to implement. Just need to change all the files that are hard-coded with absolute path assuming the app lives under the `/` directory, and tests to make sure all the responses return references to URL that are correctly pre-pended with the "directory".
